spec/filters/environment_spec.rb

```diff
@@ -21,23 +21,4 @@
       insist { subject["@metadata"]["newfield"] } == "hello world"
     end
   end
-
-  describe "does nothing on non-matching events" do
-    # The logstash config goes here.
-    # At this time, only filters are supported.
-    config <<-CONFIG
-      filter {
-        environment {
-          type => "foo"
```

Comment:
I disagree here, we have test in the code covering conditionals so I think it's okay to remove them since we delegate the responsability.